ticketutil/bugzilla.py

```diff
@@ -1,7 +1,10 @@
+import base64
 import logging
 
 import requests
 
+import mimetypes
```

Comment:
According to PEP8, because mimetypes is a built-in python module, it should be in the same block of imports as base64 and logging, in alphabetical order. So, please move `import mimetypes` to the line below `import logging`.